mgmt/rest/v2/config.go

```diff
@@ -159,3 +211,76 @@ func getPluginType(t string) (core.PluginType, error) {
 	}
 	return ityp, nil
 }
+
+// deletePluginConfigItemHelper builds different forms of request data into the way method deletePluginConfigItem accepts.
+// currently it accepts go-swagger client, swagger-ui and SanpCLI.
+func deletePluginConfigItemHelper(r *http.Request) ([]string, error) {
+	buf, err := ioutil.ReadAll(r.Body)
+	if err != nil {
+		return nil, err
+	}
+	r.Body = ioutil.NopCloser(bytes.NewBuffer(buf))
+
+	dm := map[string][]string{}
+	err = json.Unmarshal(buf, &dm)
+
+	sw := true
+	// No error needs to be returned here.
+	// As it explores different request formats.
+	// There is no way to detect a string is URLEncoded.
+	if err != nil {
+		sw = false
+
+		// go-swagger sends url-encoded form data.
+		// Unescaping is necessary.
+		data, _ := url.QueryUnescape(string(buf))
```

Comment:
@lmroz, data comes in from b.Body in URLEncoded format. That's why it unescapes it. URL object seems not going to help this case. Please let me know if you think differently.